### ISSUE TITLE
glslangValidator generates invalid SPIR-V

### ISSUE THREAD

**Issue by VadikLeshy** [KhronosGroup/glslang#2509]
glslang version: 11.1.0 either master
shaders sample: https://gitlab.com/KawaiiGraphics/KawaiiGenerated_GLSL

```
glslangValidator -V -o ./frag.spv ./*.frag.glsl
./FragmentMain.frag.glsl
./LampSignatures_PBR.frag.glsl
./PbrIlluminationCore.frag.glsl
./PBR_illumination.frag.glsl
```
```
spirv-val ./frag.spv 
error: line 491: ID 70[%normal_frag] has not been defined
  OpStore %normal_frag %225

```
SIGSEGV when attempt to use resulting SPIR-V in vulkan

**Comment by VadikLeshy** [comment]
This pull request fixes the issue for me: #2458

**Comment by greg-lunarg** [comment]
#2458 merged
